config/hydroshare-config.yaml

```diff
@@ -2,6 +2,7 @@
 HS_PATH: /home/hydro/hydroshare
 HOST_SSL_DIR: /home/hydro/hs-certs
 HS_DATABASE: pg.development.sql
+HS_LOG_FILES: /home/hydro/log
```

Comment:
Is it possible to make these relative to the local or project directory? Not all of us have a `hydro` user or `/home/hydro/` directory.